vignettes/VectorControl.Rmd

```diff
@@ -35,97 +35,156 @@ starting_EIR <- 50
+       xlab = 'Timestep (days)', ylab = expression(paste(italic(Pf),"PR"[2-10])),
+       xaxs = "i", yaxs = "i", bty = "l")
 }
 
 add_intervention_lines <- function(plot, events) {
-  plot + geom_vline(
-    data = events,
-    mapping = aes(xintercept=timestep),
-    color="blue"
-  ) + geom_text(
-    data = events,
-    mapping = aes(x = timestep, y = 0, label = name),
-    size = 4,
-    angle = 90,
-    vjust = -0.4,
-    hjust = 0
-  )
+  plot
+  abline(v = events$timestep, col = 'blue')
+  text(x = events$timestep-20, y = 0.018, events$name,
+       cex = 1, col = 'black', srt = 90)
 }
 ```
 
 Then we can run the simulation for a variety of vector control strategies:
 
-## Bed nets
-
-We can distribute bed nets once a year for two years, changing the
-characteristics of the bed nets on each distribution...
+## Bednets
 
+We can distribute bednets once a year for two years, changing the characteristics of the bed nets on each distribution.
+  
 ```{r}
 bednetparams <- simparams
 
 bednet_events = data.frame(
-  timestep = c(1, 2) * year,
-  name=c("Bednets 1", "Bednets 2")
+  timestep = c(1, 2) * year,       # The bednets will be distributed at the end of the first and the second year. 
+  name=c("Bednets 1", "Bednets 2") # These are labels for the two distributions. 
 )
 
 bednetparams <- set_bednets(
   bednetparams,
-  timesteps = bednet_events$timestep,
-  coverages = c(.8, .8),
-  retention = 5 * year,
-  dn0 = matrix(c(.533, .45), nrow=2, ncol=1),
-  rn = matrix(c(.56, .5), nrow=2, ncol=1),
-  rnm = matrix(c(.24, .24), nrow=2, ncol=1),
-  gamman = rep(2.64 * 365, 2)
+  timesteps = bednet_events$timestep, 
+  coverages = c(.8, .8),  # Each round is distributed to 80% of the population 
+  retention = 5 * year, # Nets are kept on average 5 years
+  dn0 = matrix(c(.533, .45), nrow=2, ncol=1), # Matrix of death probabilities for each mosquito species over time
+  rn = matrix(c(.56, .5), nrow=2, ncol=1), # Matrix of repelling probabilities for each mosquito species over time
+  rnm = matrix(c(.24, .24), nrow=2, ncol=1), # Matrix of minimum repelling probabilities for each mosquito species over time
+  gamman = rep(2.64 * 365, 2) # Vector of bednet half-lives for each distribution timestep
 )
 
 output <- run_simulation(sim_length, bednetparams)
 
 add_intervention_lines(plot_prevalence(output), bednet_events)
 ```
+  
+
+It is important to understand the difference between the input `coverages` argument of `malariasimulation::set_bednets()` and the resulting population bednet usage over time in the model. In the above example, bednets are distributed to 80% of the population at each distribution round. 
+  
+The average population bednet usage will be influenced by: 
+* The size and frequency of distributions specified in `set_bednets()`
+* The assumed net retention half life
+* Correlations in the recipients of nets between rounds
+  
+### Visualising usage over time
+```{r}
+output$prop_use_net <- output$n_use_net / output$n_0_100
```

Comment:
This seems to be [failing](https://github.com/mrc-ide/malariasimulation/actions/runs/4176326168/jobs/7232463459#step:9:44) on all machines with the following error message:

`replacement has 0 rows, data has 1095`

do you have `n_0_100` in the `output` dataframe? It looks like we're only checking clinical incidence between 2 and 10 years, i.e. `n_730_36500`.

Maybe you would like to divide by `human_population`?

Reply:
Thanks Giovanni - I did want to do that. I had made these changes on my local machine and thought I committed and pushed them to my vignettes/vectorcontrol branch and didn't mean to add to this PR. Apologies for my confusion! I think I will close this pull request